internal/xds/bootstrap/bootstrap.go

```diff
@@ -415,6 +418,14 @@ func NewConfigFromContents(data []byte) (*Config, error) {
 }
 
 func newConfigFromContents(data []byte) (*Config, error) {
+	// Normalize the input configuration.
```

Comment:
Comments should generally say "why" not "what" -- so why are we doing this step now?